draft-iab-protocol-maintenance.md

```diff
@@ -573,7 +573,7 @@ This document has no IANA actions.
 Internet Architecture Board members at the time this document was approved
 for publication were:
 
-<ul empty="true" spacing="compact">
+<ul empty="true" spacing="compact" bare="false" indent="3">
```

Comment:
`bare` is false by default, and I don't like specifying format like that for `indent`.  I'd prefer to leave this alone and suffer the divergence.

Reply:
Fine by me, reverted this line